keyboards/cannonkeys/bakeneko65_iso_hs/info.json

```
@@ -0,0 +1,106 @@
+{
+    "keyboard_name": "Bakeneko65 ISO HS",
+    "manufacturer": "CannonKeys",
+    "url": "https://cannonkeys.com",
+    "maintainer": "awkannan",
+    "usb": {
+        "vid": "0xCA04",
+        "pid": "0x001F",
+        "device_version": "0.0.1"
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["A8", "C13", "B9", "B8", "B7", "B6", "B5", "B4", "B3", "A7", "A5", "A4", "A3", "A2", "A1"],
+        "rows": ["A14", "A15", "A0", "B1", "B0"]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "backlight": true
+    },
+    "backlight": {
+        "breathing": true,
+        "levels": 15,
+        "pin": "A6"
+    },
+    "processor": "STM32F072",
+    "bootloader": "stm32-dfu",
+    "layouts": {
+        "LAYOUT": {
```

Same thing as [my comment](https://github.com/qmk/qmk_firmware/pull/20130#discussion_r1137594864) on other PR. Perhaps [`65_iso`](https://github.com/qmk/qmk_firmware/tree/master/layouts/default/65_iso) or [`65_iso_blocker`](https://github.com/qmk/qmk_firmware/tree/master/layouts/default/65_iso_blocker)?
```suggestion
    "layouts": {
        "LAYOUT": {
```